### ISSUE TITLE
Add fast travel feature: labeling a point as local

### ISSUE THREAD

**Issue by LevFendi** [Factorio-Access/FactorioAccess#141]
* Every point can be labelled as local or global.
* A local point is removed from the fast travel list unless it is within a range of X tiles from the player character. 
* The value for X is 50 by default but the player is able to set a different number for every point.

**Comment by ahicks92** [comment]
yes.  Though I do still want a tree at some point because I am mr. push the limits.  But this would solve everything to the end of a vanilla run and in fairness I've not done more than that as of yet (also it has the advantage of being simple to explain to users).

**Comment by ahicks92** [comment]
Never mind on the above.  Solutions for that are in #141 because we are more granular than I thought. Sorry.
